<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>241</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>320</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="bmo.qrc">:/page/image/bmosmile.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>340</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>480</width>
        <height>320</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(197, 231, 211)</string>
     </property>
     <widget class="QPushButton" name="setting">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>setting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_main">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>95</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QPushButton" name="game1">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>game1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="game2">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>game2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="game3">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>game3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage3">
     <property name="font">
      <font/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(197, 231, 211)</string>
     </property>
     <widget class="QPushButton" name="back_manu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NanumGothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>95</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QPushButton" name="exit">
         <property name="font">
          <font>
           <family>NanumGothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restart">
         <property name="font">
          <font>
           <family>NanumGothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="bmo.qrc"/>
 </resources>
 <connections/>
</ui>
